docs/jwst/outlier_detection/outlier_detection_step.rst

```diff
@@ -8,6 +8,13 @@ detection on JWST observations.  The ``OutlierDetectionStep`` supports multiple
 algorithms and determines the appropriate algorithm for the type of observation
 being processed.  This step supports:
 
+* **Image modes**: 'NRC_IMAGE', 'MIR_IMAGE', 'NRS_IMAGE', 'NIS_IMAGE', 'FGS_IMAGE'
```

Comment:
No plans to do level-3 processing of NRS_IMAGE (it's not a supported science mode - basically engineering only), so you can remove that one.